packages/cli/utils/createDir.js

```diff
@@ -0,0 +1,10 @@
+import fs from 'fs';
+import kleur from 'kleur';
+
+export async function createDir(path) {
+  await fs.mkdir(path, e => {
+    if (e) {
+      console.error(kleur.red(`Could not create file ${path}\n${e}`));
+    }
```

Comment:
I'd recommend throwing errors with the messages to display and catching them in one place.